### PR TITLE
MySQL Greatest() function added

### PR DESCRIPTION
**Pull request by shibby**
Greatest function is reverse of Least function. So, i just copied least function and changed it as `greatest` and wrote tests.

### CODE REVIEW

File: tests/Query/Mysql/StringTest.php [hunk 1]
```diff
@@ -112,6 +112,39 @@ public function testLeast()
             "SELECT LEAST('M', 'N', 'o', 'c', 'C') AS sclr_0 FROM Blank b0_",
             $q->getSql()
         );
+
+        $q = $this->entityManager->createQuery("SELECT LEAST(b.id, 15) AS lest FROM DoctrineExtensions\Tests\Entities\Blank b");
```
Comment (shibby) [inline tests/Query/Mysql/StringTest.php]: this is new assertion for testLeast function. FYI

**Comment by stevelacey** [comment]
Thanks for your contribution! Master has moved ahead since you wrote this and there are now conflicts, can you rebase and I'll get this merged in.

```
git rebase master
git push -f
```

:smile:

**Comment by shibby** [comment]
@stevelacey i think i resolved conflicts, but i don't know is that true way :)

**Comment by EmanueleMinotto** [comment]
@stevelacey @shibby any news about this?

**Comment by shibby** [comment]
@stevelacey @EmanueleMinotto 
fixed conflicts again. after test passes, it can be merged.

**Comment by stevelacey** [comment]
thanks for your contribution :smile:

**Pull request merged**